fluent-kubernetes-loomsystems.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: fluentd-loomsystems
+  namespace: kube-system
+  labels:
+    k8s-app: fluentd-loomsystems
+    version: v1
+    kubernetes.io/cluster-service: "true"
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: fluentd-loomsystems
+        version: v1
+        kubernetes.io/cluster-service: "true"
+    spec:
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: fluentd
+        image: loomsystems/fluentd-kubernetes-loomsystems:v0.12.43-loomsystems
+        imagePullPolicy: Always
+        env:
+        - name:  LOOM_RECIEVER_HOST
+          value: "{{RECEIVER HOST}}"
```

Comment:
If these are meant to be set by the user we should mention in the docs that they are expected to set these.